ci/system_integration_test.d

```diff
@@ -65,6 +78,15 @@ private int main (string[] args)
     // as the docker-compose bind volumes
     runCmd(Cleanup);
 
+    // Generate TLS files
+    foreach (dir; NodeDirs)
+    {
+        immutable key_file = dir ~ "agora-key.pem";
+        immutable cert_file = dir ~ "agora-cert.pem";
+        runCmd(["rm", "-f", cert_file, key_file]);
+        runCmd(GenCerts ~ ["-keyout", key_file, "-out", cert_file,]);
+    }
+
```

Comment:
Why not just generate it and `git add` it ?
This way, one can run the integration test manually (`docker compose up`) without using this file.